openhcl/page_pool_alloc/src/lib.rs

```diff
@@ -418,7 +418,7 @@ impl Drop for PagePoolHandle {
 /// [`PagePoolAllocatorSpawner::allocator`].
 ///
 /// This struct is considered the "owner" of the pool allowing for save/restore.
-#[derive(Inspect)]
+#[derive(Inspect, Clone)]
```

Comment:
We don't want to allow clone here - this type was explicitly not clone because we want to enforce there is only a single instance of it. Since we're taking ownership of a range of memory, it's important we can't have multiple users of that memory or it would lead to corruption. 

I haven't reviewed the rest of the PR but why did we need to add clone? The idea should be that `dma_manager` should probably take overall ownership of this, but for now perhaps it can get away with just getting an allocator spawner if you're not ready to do that level of refactoring.